calabash/Classes/FranklyServer/Routes/LPBackdoorRoute.m

```diff
@@ -29,40 +34,41 @@ - (NSDictionary *) JSONResponseForMethod:(NSString *) method URI:(NSString *) pa
     selectorName = [selectorName stringByAppendingString:@":"];
   }
 
+  id argument = [data objectForKey:@"arg"];
+  if (!argument) {
+    LPLogError(@"Expected data dictionary to contain an 'arg' key");
```

Comment:
Hmm... this creates an ambiguous case. Treating all `[NSNull null]` as `nil` would mean that users can't invoke using literal `null`, which could be problematic if the user was trying to insert values into a dict or something. 

But treating all `[NSNull null]` as proper `null` will lead to NPEs if the user is expecting it to be treated as `nil`. 

I can imagine a few ways around this, but they're awkward and require more changes than we'd probably want to deal with now. 
E.g.:

```
//User wants to invoke a selector with sig:
// - (id)foo:(id)foo bar:(id)bar baz:(id)baz;
{
    @"method" : @"foo:bar:baz:",
    @"arguments" : @{
        @"foo:" : @"a value",
        @"baz" : [NSNull null]
    }
}

//The method would be invoked with:
[target foo:@"a value" bar:nil baz:[NSNull null]];
```